crates/db_views_actor/src/structs.rs

```diff
@@ -120,3 +120,16 @@ pub struct PersonView {
   pub person: Person,
   pub counts: PersonAggregates,
 }
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
+#[cfg_attr(feature = "full", derive(TS, Queryable))]
+#[cfg_attr(feature = "full", ts(export))]
+/// Followed community posts view.
+pub struct FollowedCommunityPostView {
+  pub community: Community,
+  pub pending: bool,
+  pub notifications_enabled: bool,
+  pub post: Post,
+  pub post_read: bool,
+}
```

Comment:
Why does this need a separate struct? Seems like the existing PostView should be enough.

Reply:
This might be dumb, but the problem was that I couldn't access `PostView` from db_views_actor since it's in a sibling crate